main.go

```diff
@@ -15,8 +15,22 @@ limitations under the License.
 */
 package main
 
-import "github.com/falcosecurity/falcoctl/cmd"
+import (
+	"os"
+
+	"github.com/falcosecurity/falcoctl/cmd"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
can we not use our client-go instead of k8s.io client ?

Reply:
What do you mean?

Reply:
?

Reply:
so right now we are using k8s github repo client-go to talking to k8s cluster , we have our own https://github.com/falcosecurity/client-go repo , can we not solely dependent on our falco client-go. otherwise everything looks good.

Reply:
Our `client-go` is to talk to Falco, on the other hand the `k8s.io/client-go` is to talk to Kubernetes, so we need both.

Reply:
okay , thanks @fntlnz .